src/fontr/fontr/transforms.py

```diff
@@ -0,0 +1,65 @@
+import torchvision.transforms as transforms
+import torch
+import random
+
+
+class Patch(torch.nn.Module):
+    """
+    Creates patches from images
+    """
+
+    def __init__(self, patch_size=10, patch_width=96, patch_height=96):
+        self.patch_size = patch_size
+        self.patch_width = patch_width
+        self.patch_height = patch_height
+
+    def __call__(self, image):
+        c, h, w = image.shape
+
+        patches = []
+
+        for _ in range(self.patch_size):
+            if w > self.patch_width:
+                left = random.randint(0, w - self.patch_width)
```

Comment:
hmm, this randomness actually makes inference non-determinant
let's leave this this way anyway; I see they don't care in the original paper